src/test/mempool_tests.cpp

```diff
@@ -523,7 +523,7 @@ BOOST_AUTO_TEST_CASE(MempoolSizeLimitTest)
     pool.addUnchecked(tx6.GetHash(), entry.Fee(1100LL).FromTx(tx6));
     pool.addUnchecked(tx7.GetHash(), entry.Fee(9000LL).FromTx(tx7));
 
-    // we only require this remove, at max, 2 txn, because its not clear what we're really optimizing for aside from that
+    // we only require this remove, at max, 2 txn, because it's not clear what we're really optimizing for aside from that
```

Comment:
nit: prefer 'we only require this to remove...' or 'we only require that this remove...'